pillar/monit/lms_503.sls

```diff
@@ -7,5 +7,5 @@ monit_app:
         with:
           address: lms.mitx.mit.edu
         if:
-          failed: port 443 protocol https status = 503
+          failed: port 443 protocol https request "/heartbeat" status = 200
```

Comment:
Does this mean that it will fail if the heartbeat endpoint returns a success?

Reply:
No my understanding of the docs is that if it doesn't return a 200, that would be considered as a failure and would trigger the alert. From the docs:
```
For example to test that a page does not exist (the HTTP server should return 404 in this case):

  if failed
     port 80
     protocol http
     request "/non/existent.php"
     status = 404
  then alert
```